### DISCUSSION TITLE
Freshers (2021 graduates) DS160 ambiguity

### DISCUSSION THREAD

**Discussion by avsr2k** [snigi-gupta/AttendingUniversityAtBuffalo#34] [Q&A] [answered]
What did you guys fill in the DS10 work/education training section. I'm in my 8th semester and my last working day of college is in June 10th and my visa interview is scheduled for June 21st, if I choose not employed i am required to enter the last date of attendance in the form, which i cannot because the latest date the form will accept is today's (current date). Should i enter student and leave it be?

**Comment by snigi-gupta** [comment, marked as answer]
Hi @avsr2k 
Firstly I apologize for the delay in my response. Lately, I have been dealing with a lot of change, hence the delay.

I think you should enter student since that is your current status while filling the form.
